src/ng/compile.js

```diff
@@ -1386,6 +1386,61 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var ignoreMDirectives = false;
+  /**
+   * @ngdoc method
+   * @name $compileProvider#commentDirectivesEnabled
+   * @description
+   *
+   * It indicates to the compiler if the current project
+   * does require to use comment directives.
+   *
+   * If comment directives are disabled
+   * it allows to the compiler to make optimizations
+   * and skip comments parsing.
+   *
+   * Default value is ‘true’
+   *
+   * Example:
+   *
+   * ```
+   * $compileProvider.commentDirectivesEnabled(false);
+   * ```
+   *
+   * @param {boolean} false if the compiler may ignore comment directives
+   */
+  this.commentDirectivesEnabled = function(value) {
+    ignoreMDirectives = !value;
+  };
+
+
+  var ignoreCDirectives = false;
+  /**
+   * @ngdoc method
+   * @name $compileProvider#cssClassDirectivesEnabled
+   * @description
+   *
+   * It indicates to the compiler if the current project
```

Comment:
The description should be something along these lines 
> Whether or not directives on element classes should be compiled. Defaults to `true`.
Calling this function with `false` disables the compilation of directives on element classes for the whole application. This results in a compilation performance gain, as the compiler doesn't have to check element classes when looking for directives. This should however only be used if you are sure that no class directives are used in the application (including any 3rd party directives).

Same change is needed for comment directives, too